slack-archive/src/main/java/juja/microservices/slack/archive/model/entity/Message.java

```diff
@@ -1,4 +1,4 @@
-package juja.microservices.slack.archive.model;
+package juja.microservices.slack.archive.model.entity;
```

Comment:
need @entity annotation

Reply:
i read spring docs. MongoTemplate don't required @Entity annotation

9.5. Saving, Updating, and Removing Documents
https://docs.spring.io/spring-data/mongodb/docs/current/reference/html/#mongo-template

and there are two ways with Id annotation. 
1. A property or field annotated with @Id (org.springframework.data.annotation.Id) will be mapped to the _id field.
2. A property or field without an annotation but named id will be mapped to the _id field.

I think it is a good case to use  @Id annotation explicitly

Reply:
I agree with Nikolay